cheroot/cli.py

```diff
@@ -228,7 +228,7 @@ def parse_wsgi_bind_addr(bind_addr_string):
         type=int,
         help='Timeout in seconds for putting requests into queue',
     ),
-}
+}  # noqa: C408
```

Comment:
This violation is related to `dict()` constructor being used internally, I don't think it's related to the literal: https://github.com/adamchainz/flake8-comprehensions#c408-unnecessary-dictlisttuple-call---rewrite-as-a-literal. Also, the preference is to fix violations, not ignore them. Ignores should only be used temporarily.

Reply:
updated